packages/terra-avatar/docs/README.md

```diff
@@ -0,0 +1,32 @@
+# Terra Avatar
+
+The terra-avatar component displays an avatar which can be either an image or text. If neither an image or text is provided, a variant-based fallback icon is used. The avatar displays in a circular wrapper.
```

Comment:
Nitpick.

Let's combine the first and third sentences. I think it will read a little better

    The terra-avatar component displays an avatar, which can be either an image or text, in a circular wrapper.

Also, I would rather say circular frame than circular wrapper. Wrapper feels like a code element while frame is a visual display.

Reply:
Updated in this [commit](https://github.com/cerner/terra-core/commit/855d5fc5c14ce5dc2a29cb07d7d5575af2320962).